README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
+
+- Automated resources for Vagrant box creation
+
+Resources which automate the Vagrant box creation process are contained here. Users can define
+box specifications based on which the Vagrant boxes are generated.
+
+- Vagrantfile for WSO2 Identity Server single node deployment with Analytics support
+
+Vagrantfile resource which performs an automated deployment of WSO2 Identity Server single node
```

Comment:
You can simply mention "Vagrantfile performs" instead of saying "Vagrant file resource which performs"

Reply:
+1